save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/security/CustomAuthenticationBasicConverter.kt

```diff
@@ -0,0 +1,35 @@
+package com.saveourtool.save.sandbox.security
```

Comment:
can we move it to common module and re-use in backend and sandbox?

Reply:
Same as above